code/game/objects/structures/grille.dm

```diff
@@ -1,3 +1,6 @@
+GLOBAL_LIST_INIT(grille_blend_objects,   list(/obj/machinery/door, /turf/simulated/wall))
+GLOBAL_LIST_INIT(grille_noblend_objects, list(/obj/machinery/door/window))
```

Comment:
Where are these used?

Reply:
Ah they aren't, I was breaking them out into globals before realizing they all use the exact same list.